src/amo/components/SearchPage.js

```
@@ -11,15 +11,19 @@ import SearchResult from './SearchResult';
 export default class SearchPage extends React.Component {
   static propTypes = {
     count: PropTypes.number,
-    lang: PropTypes.string.isRequired,
+    lang: PropTypes.string,
     loading: PropTypes.bool.isRequired,
     page: PropTypes.number,
+    params: PropTypes.object(PropTypes.shape({
+      lang: PropTypes.string.isRequired,
+    })),
     results: PropTypes.array,
     query: PropTypes.string,
   }
 
   render() {
-    const { count, lang, loading, page, query, results } = this.props;
+    const { count, loading, page, query, results } = this.props;
+    const lang = this.props.params ? this.props.params.lang : this.props.lang;
```

Should this throw if you got here and it was still undefined?